python/cudf/cudf/_lib/copying.pyx

```diff
@@ -765,7 +765,7 @@ cdef class _CPackedColumns:
         gpu_data = Buffer.deserialize(header["data"], frames)
 
         dbuf = DeviceBuffer(
-            ptr=gpu_data.ptr,
+            ptr=gpu_data.get_ptr(mode="write"),
```

Comment:
Ah wait, no, in the `SpillableBuffer` case we don't want to mark these data as unspillable, no? Although I guess it doesn't matter because the `gpu_data` object doesn't persist past the end of this function (unless it's tracked somewhere globally?), so what we're doing here is transferring ownership of the pointer?